src/number/n_Z.jl

```diff
@@ -357,6 +357,17 @@ end
 
 promote_rule(C::Type{n_Z}, ::Type{T}) where {T <: Integer} = n_Z
 
+BigInt(n::n_Z) = libSingular.n_GetMPZ(n.ptr, parent(n).ptr)
```

Comment:
I doubt you can have Julia clean up a BigInt that Singular allocated. This will probably result in crashes.

Reply:
I just re-used the pre-existing definition for `convert(::Type{BigInt}, n::n_Z)`. Actually, in `n_GetMPZ`, the resulting `BigInt` is allocated on the Julia side (cf. https://github.com/oscar-system/Singular.jl/blob/796dbc9c0293b6dd004e17fb3898570b290d18b7/src/libsingular/coeffs.jl#L8).